src/presolve.jl

```diff
@@ -723,3 +728,33 @@ function remove_dominated_columns!(ps::PresolveData{T}) where {T}
     end
     return nothing
 end
+
+"""
+    coefficient_strengthening!(ps::PresolveData)
+
+Perform coefficient strengthening for integer/binary variables
+on every constraints.
+
+Called once at the beginning of the presolve procedure.
+"""
+
+function coefficient_strengthening!(ps::PresolveData{T}) where {T}
```

Comment:
This function can go in `mip/coefficient_strengthening.jl`.

Putting everything in `presolve.jl` was not a good practice of mine :)